nilearn/plotting/surf_plotting.py

```diff
@@ -527,8 +527,8 @@ def _plot_surf_matplotlib(coords, faces, surf_map=None, bg_map=None,
                           alpha='auto', bg_on_data=False,
                           darkness=.7, vmin=None, vmax=None, cbar_vmin=None,
                           cbar_vmax=None, cbar_tick_format='%.2g',
-                          title=None, title_font_size=18, output_file=None,
```

Comment:
from my understanding the `title_font_size` was passed to mirror what is being done with plotly, but it looks like it never got implemented in the matplotlib side

so maybe this PR should try to get to behavior parity between matplotlib and plotly rather than removing this unused parameter: maybe better in a side PR?

Reply:
@bthirion what's your take on this?